scripts/run-gui-backend

```diff
@@ -0,0 +1 @@
+run-gui-backend.guiweston
```

Comment:
Looks good to me. I just wonder, shouldn't the default be the original. It shouldn't matter but seems more robust to me because `weston` is used only by one spin and second solution by everyone else.

Reply:
Weston is the reference compositor and is what I would call a reasonable Wayland choice if nobody has an opinion on compositors. 😉

Reply:
If I understand it correctly then it's question Weston vs. others because everything else is the original script.
In that case I don't think Weston should be taken as default from by what it is used.

Reply:
Well, Weston makes sense for boot.iso and install DVD because it would be able to replace your existing infrastructure for graphics (on-screen, VNC, and add RDP). It also would be useful so that if you wanted to drop "support" for running on pure X11, even for X11 spins, this would still work. 

Making this configurable ensures that spins with Wayland compositors can choose their compositor to run it if it is capable of doing so.

Reply:
Weston is unfortunately not supported by all our deliverables(RHEL, CentOS and derivates). So at the end it would add support complexity if it is used as default.